app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/FileContainer.kt

```diff
@@ -0,0 +1,63 @@
+package com.hedvig.android.design.system.hedvig
+
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.dp
+import coil.ImageLoader
+import coil.compose.AsyncImage
+import coil.compose.AsyncImagePainter
+import coil.request.ImageRequest
+import com.hedvig.android.placeholder.PlaceholderHighlight
+import com.hedvig.android.placeholder.fade
+import com.hedvig.android.placeholder.placeholder
+
+@Composable
+fun FileContainer(model: Any, imageLoader: ImageLoader, modifier: Modifier = Modifier, cacheKey: String? = null) {
```

Comment:
If this is only used from inside the DS module for now, we should keep it internal